docs/rule/sort-invocations.md

```diff
@@ -0,0 +1,113 @@
+# sort-invocations
+
+🔧 The `--fix` option on the command line can automatically fix some of the problems reported by this rule.
+
+1. [Why use it?](#why-use-it)
+1. [Examples](#examples)
+    - [Components](#components)
+    - [Helpers](#helpers)
+    - [Modifiers](#modifiers)
+1. [Configuration](#configuration)
+1. [Known issues](#known-issues)
+1. [References](#references)
+
+
+## Why use it?
+
+The rule helps you standardize templates:
+
+- Component invocations
+- Helper invocations
+- Modifier invocations
+
+By sorting things that are order-independent, you can more easily refactor code. In addition, sorting removes style differences, so you can review another person's code more effectively.
+
+> [!TIP]
+> 
+> The `--fix` option for this rule doesn't preserve formatting. To preserve formatting, you can use [`ember-template-lint-plugin-prettier`](https://github.com/ember-template-lint/ember-template-lint-plugin-prettier) and [`prettier-plugin-ember-template-tag`](https://github.com/ember-tooling/prettier-plugin-ember-template-tag) to format `*.hbs` and `*.{gjs,gts}`, respectively.
+
+
+## Examples
+
+### Components
+
+When invoking a component, list things in the following order:
+
+1. Arguments
+2. Attributes
+3. Modifiers
+4. Splattributes
+
+The order clearly shows how the component is customized more and more. Things are alphabetized within each group.
+
+```hbs
+<Ui::Button
+  @label="Submit form"
+  @type="submit"
+  data-test-button
+  {{on "click" this.doSomething}}
+  ...attributes
+/>
+```
+
+
+### Helpers
+
+When invoking a helper, list the named arguments in alphabetical order.
+
+```hbs
+{{t
+  "my-component.description"
+  installedOn=this.installationDate
+  packageName="@ember/source"
+  packageVersion="6.0.0"
+}}
+```
+
+
+### Modifiers
+
+Similarly to helpers, list the named arguments in alphabetical order.
+
+
+## Limitations
+
+It's intended that there are no options for sorting. Alphabetical sort is the simplest for everyone to understand and to apply across different projects. It's also the easiest to maintain.
+
+To better meet your needs, consider creating a plugin for `ember-template-lint`.
```

Comment:
The same approach (limit the API) is taken by [`eslint-plugin-simple-import-sort`](https://github.com/lydell/eslint-plugin-simple-import-sort/tree/v12.1.1?tab=readme-ov-file#not-for-everyone).

Reply:
The counter to that approach is https://github.com/import-js/eslint-plugin-import